lib/sync-server.js

```diff
@@ -154,7 +154,7 @@ function start(cb) {
   syncStarted = true;
 
   if (mongoDbClient === null || redisClient === null) {
-    throw new Error('MongoDB Client & Redis Client are not connected. Ensure connect() is called before calling start');
+    throw new Error('MongoDB Client & Redis Client are not connected. Ensure you have called sync.connect() before calling sync.init()');
```

Comment:
When I updated from `fh-mbaas-api@6` to `fh-mbaas-api@7` I kept seeing this error, but it wasn't 100% clear what the problem was. I think adding the `sync` prefix and saying `init` instead of `start` will help clear up some confusion for those who try upgrade.